scripts/create-component.js

```diff
@@ -13,7 +13,7 @@ const statelessSequence = ['GlobalIndex', 'Styles', 'Index', 'View', 'ViewTest',
 const rootComponentFolderPath = './packages/experiments/src/';
 const componentFolderPath = rootComponentFolderPath + 'components/' + newComponentName + '/';
 const componentPathNamePrefix = rootComponentFolderPath + 'components/' + newComponentName + '/' + newComponentName;
-const templateFolderPath = './scripts/templates';
+const templateFolderPath = './scripts/templates/create-component';
```

Comment:
@JasonGore Heads up that I'm moving the component and package templates into subfolders--let me know if that's an issue